cfbs/commands.py

```diff
@@ -49,6 +50,10 @@
 from cfbs.module import Module
 
 
+class InputDataUpdateFailedException(Exception):
```

Comment:
I think the `Exception` part is not needed in the name of the exception class. If you want to have something there, `Error` is usually used.